pkg/errors/errors.go

```diff
@@ -446,6 +446,10 @@ func (e *ServiceError) IsFailedToCheckQuota() bool {
 	return e.Code == FailedToCheckQuota("").Code
 }
 
+func (e *ServiceError) IsInstanceTypeNotSupported() bool {
+	return e.Code == InstanceTypeNotSupported("").Code
```

Comment:
Instead of instantiating InstanceTypeNotSupported, I think we can check the code directly:
```suggestion
	return e.Code == ErrorInstanceTypeNotSupported
```